ui/src/overview.tsx

```diff
@@ -291,3 +309,204 @@ function ParticipantCriteria(props: {
     </Grid>
   );
 }
+
+type DemographicChartsProps = {
+  cohort: tanagra.Cohort;
+  underlayName: string;
+};
+
+function DemographicCharts({ cohort, underlayName }: DemographicChartsProps) {
+  const api = useContext(EntityCountsApiContext);
+
+  const barColours = [
+    "#003f5c",
+    "#2f4b7c",
+    "#665191",
+    "#a05195",
+    "#d45087",
+    "#f95d6a",
+    "#ff7c43",
+    "#ffa600",
+  ];
+
+  const fetchDemographicData = useCallback(async () => {
+    const searchEntityCountsRequest: tanagra.SearchEntityCountsRequest = {
+      entityCounts: {
+        entityVariable: "p",
+        additionalSelectedAttributes: ["gender", "race"],
+        groupByAttributes: [
+          "gender_concept_id",
+          "race_concept_id",
+          "year_of_birth",
+        ],
+        filter: generateQueryFilter(cohort, "p"),
+      },
+    };
+
+    const data = await api.searchEntityCounts({
+      underlayName: underlayName,
+      entityName: "person",
+      searchEntityCountsRequest: searchEntityCountsRequest,
+    });
+
+    if (!data.counts) {
+      throw new Error(
+        "The counts property returned by the searchEntityCounts API is undefined."
+      );
+    }
+
+    const demographicData = data.counts;
+    let totalCount = 0;
+    const demographicsByGender = new Map();
```

Comment:
A follow up task for this will be to make this component generic rather than hard-coding specific properties. That will at least need to wait until my current PR is submitted but you could start thinking about how to accomplish it. It would be along the lines of how the `CriteriaConfigs` work.